app/views/provider/admin/dashboards/_developers_navigation.html.slim

```diff
@@ -53,13 +53,13 @@ nav.DashboardNavigation
     // Messages
     li.DashboardNavigation-list-item
       = dashboard_collection_link 'Message',
-                                  current_account.messages.where(sender: !current_account),
+                                  current_account.received_messages.not_system,
```

Comment:
why not system? Looks great otherwise

Reply:
`not_system` are messages that have `system_operation_id = NULL`, and these are messages that are actually sent by users to other users (developer or provider).

Those that are "system" are just notifications on some events that happen (e.g. account signed up, new application created etc., full list [here](https://github.com/3scale/porta/blob/553d28020f4128155ba523d216aac13555d9e3c6/app/models/system_operation.rb#L8-L23))